### PR TITLE
chore: add meta attribute to native

### PR DESCRIPTION
**Pull request by chaitanyadeorukhkar**
Adds meta attribute to native, similar to web. We need this for blade coverage tracking on native

### CODE REVIEW

File: packages/blade/src/utils/metaAttribute/metaAttribute.native.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+
+const metaAttribute = ({
+  testID,
+  name,
+}: {
+  name?: string;
+  testID?: string;
+}): Record<string, string> => {
   return {
+    ...(name ? { [`data-${MetaConstants.Component}`]: name } : {}),
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/utils/metaAttribute/metaAttribute.native.ts:11]: kept this to maintain parity with web but we won't be using this.

**Comment by saurabhdaware** [comment]
Conflicts 🙈

File: packages/blade/src/utils/metaAttribute/metaAttribute.native.ts [hunk 2]
```diff
@@ -1,5 +1,14 @@
-const metaAttribute = ({ testID }: { name?: string; testID?: string }): Record<string, string> => {
+import { MetaConstants } from './metaConstants';
+
+const metaAttribute = ({
```
Comment (divyanshu013) [inline packages/blade/src/utils/metaAttribute/metaAttribute.native.ts:3]: Minor: might be good to have some unit tests for this util

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

File: packages/blade/src/components/Box/__tests__/Box.native.test.tsx [hunk 3]
```diff
@@ -17,6 +17,7 @@ describe('<Box />', () => {
     );
     expect(toJSON()).toMatchInlineSnapshot(`
       <View
+        data-blade-component="box"
```
Comment (kamleshchandnani) [inline packages/blade/src/components/Box/__tests__/Box.native.test.tsx:20]: does this works on native? IIRC from past, custom attributes were discarded by RN?

Comment (chaitanyadeorukhkar) [reply packages/blade/src/components/Box/__tests__/Box.native.test.tsx:20]: It doesn't discard it during tests. So the react-test-renderer that's used during tests to generate these snapshots doesn't discard it.

**Review by divyanshu013 — APPROVED** [review]
:shipit:

**Pull request merged**